packages/frontend2/src/app/_components/table/cells/index-cell.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import { cn } from '~/utils/cn'
+
+interface IndexCellProps {
+  index: number
+  className?: string
+}
+
+export function IndexCell({ index, className }: IndexCellProps) {
+  return (
+    <span
+      className={cn(
+        'ml-auto tabular-nums font-medium text-gray-50 text-xs dark:font-normal dark:text-gray-600',
+        className,
+      )}
+    >
+      {index + 1}
```

Comment:
Can we do sth about this? I don't really get the idea of supplying an index just to display `index + 1` here if the component is named IndexCell.